icecreamshop/icecreamshop/src/main/java/com/summerschool/icecreamshop/model/Basket.java

```diff
@@ -0,0 +1,30 @@
+package com.summerschool.icecreamshop.model;
+import java.util.List;
+
+public class Basket {
+    private Integer id;
+    private String sessionid;
+    private List<BasketProduct> basketProductList;
+    public Integer getId() {
+        return id;
+    }
+
+    public void setId(Integer id){
+        this.id=id;
+    }
+
+    public String getSessionid(){
+        return sessionid;
+    }
+
+    public void setSessionid(String sessionid){
+        this.sessionid=sessionid;
+
+    }
+
+     Basket b= new Basket(Integer id, String sessionid){
+        this.id=id;
+        this.sessionid=sessionid;
+     }
```

Comment:
constructorul ar trebui schimbat aici, trebuie sa arate ca o metoda, cu aceeasi parametri si continut intre {} dar in loc de `Basket b= new Basket` ar trebui public Basket(Integer id, String sessionid) { ce ai acum }

Reply:
te poti inspira din constructorul din clasa de BasketProduct pentru a-l face pe asta